airbyte-integrations/connectors/source-wordpress/manifest.yaml

```diff
@@ -0,0 +1,4772 @@
+          page_token_option:
+            type: RequestOption
+            field_name: page
+            inject_into: request_parameter
+          pagination_strategy:
+            type: PageIncrement
+            page_size: 100
+            start_from_page: 1
+            inject_on_first_request: true
+        requester:
+          $ref: "#/definitions/base_requester"
+          path: wp/v2/categories
+          http_method: GET
+        record_selector:
+          type: RecordSelector
+          extractor:
+            type: DpathExtractor
+            field_path: []
+      primary_key:
+        - id
+      schema_loader:
+        type: InlineSchemaLoader
+        schema:
+          $ref: "#/schemas/categories"
+    taxonomies:
+      type: DeclarativeStream
+      name: taxonomies
+      retriever:
+        type: SimpleRetriever
+        paginator:
+          type: DefaultPaginator
+          page_token_option:
+            type: RequestOption
+            field_name: page
+            inject_into: request_parameter
+          pagination_strategy:
+            type: PageIncrement
+            page_size: 100
+            start_from_page: 1
+            inject_on_first_request: true
+        requester:
+          $ref: "#/definitions/base_requester"
+          path: wp/v2/taxonomies
+          http_method: GET
+        record_selector:
+          type: RecordSelector
+          extractor:
+            type: DpathExtractor
+            field_path: []
+      primary_key:
+        - category
+      schema_loader:
+        type: InlineSchemaLoader
+        schema:
+          $ref: "#/schemas/taxonomies"
+  base_requester:
+    type: HttpRequester
+    url_base: https://{{ config['domain'] }}/wp-json/
+    authenticator:
+      type: BasicHttpAuthenticator
+      password: "{{ config[\"password\"] }}"
+      username: "{{ config[\"username\"] }}"
+
+streams:
+  - $ref: "#/definitions/streams/users"
+  - $ref: "#/definitions/streams/posts"
+  - $ref: "#/definitions/streams/categories"
+  - $ref: "#/definitions/streams/plugins"
+  - $ref: "#/definitions/streams/blocks"
+  - $ref: "#/definitions/streams/comments"
+  - $ref: "#/definitions/streams/pages"
+  - $ref: "#/definitions/streams/tags"
+  - $ref: "#/definitions/streams/revisions"
+  - $ref: "#/definitions/streams/media"
+  - $ref: "#/definitions/streams/taxonomies"
+  - $ref: "#/definitions/streams/types"
+  - $ref: "#/definitions/streams/themes"
+  - $ref: "#/definitions/streams/statuses"
+  - $ref: "#/definitions/streams/search"
+  - $ref: "#/definitions/streams/settings"
+
+spec:
+  type: Spec
+  connection_specification:
+    type: object
+    $schema: http://json-schema.org/draft-07/schema#
+    required:
+      - domain
+      - password
+      - username
+      - parent_id
+    properties:
+      domain:
+        type: string
+        description: The domain of the WordPress site
+        name: domain
+        order: 0
+        title: Domain
+        default: http://biplab.great-site.net
```

Comment:
As far I know, Airbyte doesn't allow connections to `http` websites in Airbyte cloud. Maybe this is the problem you're facing.

Reply:
@marcosmarxm yes I also discovered that when I was using the mentioned domain here but I changed it to [this](https://blanchedalmond-meerkat-467322.hostingersite.com/) . I dont know why it isn't reflecting here though Its there in the UI builder. You can confirm that from the screenshot in the chat above.